recipes/dionysus/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dionysus" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "78c24da06fd273f0103ad4cb86331f5f4518d7f0445e2cecd0e325ca1a2d011f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cmake
+    - boost
+  run:
+    - python
+
+test:
+  imports:
+    - dionysus
+
+about:
+  home: http://mrzv.org/software/dionysus2
+  license: BSD 3-Clause
```

Comment:
One more nitpick I missed earlier: should be `BSD-3-Clause`, with the extra dash.